medicai/layers/conv.py

```
@@ -166,17 +166,17 @@ class DepthwiseConv3D(layers.Conv3D):
                 depth_multiplier=2
             )(x)
             print(y.shape) # (1, 16, 64, 64, 6)
-    
+
     Returns:
-        ``keras.KerasTensor``: Output tensor of shape 
+        ``keras.KerasTensor``: Output tensor of shape
         ``(batch, d_out, h_out, w_out, input_channels × depth_multiplier)``,
```

Ruff (0.16.3)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)